src/server/qgsconfigcache.cpp

```diff
@@ -47,6 +47,10 @@ const QgsProject *QgsConfigCache::project( const QString &path )
       mProjectCache.insert( path, prj.release() );
       mFileSystemWatcher.addPath( path );
     }
+    else
+    {
+      QgsMessageLog::logMessage( "Error, project file '"  + path + "': " + prj->error(), QStringLiteral( "Server" ), Qgis::Critical );
```

Comment:
Better translate this and use a template based approach.

`tr( "Error when loading project file '%1': %2 " ).arg( ... )`

Reply:
I don't see any translated log message on QGIS server, and I think that better to keep it like it...

Reply:
I think that error messages should be translated, but not OGC exceptions (defined as is in the standard).

Reply:
OK..., Than I will do it :-)

Reply:
Updated :-)

Reply:
Thanks. Now for the fun part, `arg()` takes multiple parameters and is more performant in this case so it's good practice to use it